packages/market-service/src/coingecko/coingecko.ts

```diff
@@ -124,13 +114,13 @@ export class CoinGeckoMarketService implements MarketService {
       Also a lot of time when max_supply is null, total_supply is the maximum supply on coingecko
       We can reassess in the future the degree of precision we want on that field */
       return {
-        price: marketData?.current_price?.[currency],
-        marketCap: marketData?.market_cap?.[currency],
+        price: bnOrZero(marketData?.current_price?.[currency]).toString(),
+        marketCap: bnOrZero(marketData?.market_cap?.[currency]).toString(),
         changePercent24Hr: marketData?.price_change_percentage_24h,
```

Comment:
Note: This one is left as the number it is coming as from the API, and would require both this line and `@shapeshiftoss/types` to be updated:
https://github.com/shapeshift/lib/blob/6274ccdd7e0b5c6190da68ff1e148a125bcf5cdb/packages/types/src/market.ts#L5